datasets/inspecties/dataset.json

```diff
@@ -0,0 +1,281 @@
+{
+  "id": "inspecties",
+  "type": "dataset",
+  "auth": "FP/MDW",
+  "authorizationGrantor": "Directie Stadsbeheer",
+  "theme": [
+    "duurzaamheid en milieu",
+    "Ruimte en Topografie"
+  ],
+  "homepage": "https://data.amsterdam.nl",
+  "owner": "Gemeente Amsterdam, Stadsbeheer",
+  "dateModified": "2021-04-27",
+  "spatialDescription": "Gemeente Amsterdam",
+  "version": "1.0.0",
+  "title": "Inspecties",
+  "language": "nl",
+  "dateCreated": "",
+  "license": "Creative Commons, Naamsvermelding",
```

Comment:
Misschien vervangen door "niet voor publicatie" oid?

Reply:
Daar heb je helemaal gelijk. Ik ga het aanpassen.

Reply:
Licentie is naar "Anders, Niet Open" aangepast